### PR TITLE
Staking rewards

### PR DESCRIPTION
**Pull request by fadeev**

### CODE REVIEW

File: docs/architecture/rewards.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+Currently, staking rewards are comprised of transaction fees collected each
+block and fixed rewards coming from the emissions pool.
+
+$$
+StakingRewards = TransactionFees + {EmissionsPool \over NumberOfBlocksPerYear × 4}
+$$
+
+Emissions pool address:
+[zeta1w43fn2ze2wyhu5hfmegr6vp52c3dgn0srdgymy](https://zetachain.blockpi.network/lcd/v1/public/cosmos/auth/v1beta1/module_accounts)
```
Comment (kingpinXD) [inline docs/architecture/rewards.md:15]: maybe we can use this query instead , https://zetachain-athens.blockpi.network/lcd/v1/public/zeta-chain/emissions/list_addresses

File: docs/architecture/rewards.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+ZetaChain is a delegated proof of stake blockchain. Holders of ZETA tokens can
+stake their tokens by delegating to validators. Staking tokens allows holders to
+participate in governance and earn rewards for securing the network.
+
+Currently, staking rewards are comprised of transaction fees collected each
+block and fixed rewards coming from the emissions pool.
+
+$$
+StakingRewards = TransactionFees + {EmissionsPool \over NumberOfBlocksPerYear × 4}
```
Comment (kingpinXD) [inline docs/architecture/rewards.md:11]: `{EmissionsPool \over NumberOfBlocksPerYear × 4}` * validator rewards percentage

File: docs/architecture/rewards.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Staking Rewards
+
+ZetaChain is a delegated proof of stake blockchain. Holders of ZETA tokens can
+stake their tokens by delegating to validators. Staking tokens allows holders to
+participate in governance and earn rewards for securing the network.
```
Comment (kingpinXD) [inline docs/architecture/rewards.md:5]: Should we mention how commissions and slashing might affect these numbers for delegators? 
We use the default distribution module, a reference link to that might work as well

File: docs/architecture/rewards.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+Emissions pool address:
+[zeta1w43fn2ze2wyhu5hfmegr6vp52c3dgn0srdgymy](https://zetachain.blockpi.network/lcd/v1/public/cosmos/auth/v1beta1/module_accounts)
+
+You can find the emissions pool balance by querying the API:
+
+https://zetachain.blockpi.network/lcd/v1/public/cosmos/bank/v1beta1/balances/zeta1w43fn2ze2wyhu5hfmegr6vp52c3dgn0srdgymy
+
+- The emission pool is currently funded manually
+- Currently, there is no ZETA token emission in the network
```
Comment (kingpinXD) [inline docs/architecture/rewards.md:22]: What does this mean? We do have emissions enabled ?

File: docs/architecture/rewards.md (hunk 1 above)
Comment (fadeev) [reply docs/architecture/rewards.md]: 2f2d580

File: docs/architecture/rewards.md (hunk 2 above)
Comment (fadeev) [reply docs/architecture/rewards.md]: 391a97d

File: docs/architecture/rewards.md (hunk 3 above)
Comment (fadeev) [reply docs/architecture/rewards.md]: 393ccbb

File: docs/architecture/rewards.md (hunk 4 above)
Comment (fadeev) [reply docs/architecture/rewards.md]: Replaced with "Currently, there is no emission of new ZETA tokens (inflation) in the network". 86bf640

Comment (kingpinXD) [reply docs/architecture/rewards.md]: Do you think it would be sufficient to say just 
`Currently, there is no inflation in the network` ?

Comment (fadeev) [reply docs/architecture/rewards.md]: 99184c3

**Pull request merged**